### PR TITLE
Content: Creating Person for BeccaBunny

### PR DESCRIPTION
**Pull request by Zitchas**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR creates the person "Prototype B3-CC4", flying a "Shooting Star" ship reminiscent of EVN's Krypt Pod.

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [X] I uploaded the necessary image, blend, and texture assets here: [zip file on google drive](https://drive.google.com/file/d/13tE6Q425FVoRdnz7dWZOCzogQGbHIq4u/view?usp=sharing)
 - [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: [link to high-dpi repository](https://github.com/endless-sky/endless-sky-high-dpi/pull/142)
  
image shows both B3-CC4 and a shooting star escort (and thus with merchant coloring)
Prototype B3-CC4 in flight

### CODE REVIEW

File: data/remnant/remnant persons.txt [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+	gun 0 0
+	turret 0 0
+	turret 0 0
+	turret 0 0
+	explode "tiny explosion" 5
+	explode "small explosion" 20
+	explode "medium explosion" 25
+	explode "large explosion" 15
+	"final explode" "final explosion medium"
+	description "This ship appears to be a hollowed-out asteroid filled with strange equipment. It is unclear what is creating the rays of light, but they clearly have something to do with how it moves without obvious engines."
```
Comment (beccabunny) [inline data/remnant/remnant persons.txt:158]:
```suggestion
	description "This ship appears to be a hollowed-out asteroid filled with strange equipment. It is unclear what is creating the rays of light, but they must have something to do with how it moves without obvious engines."
```
a derivation of "clear" was used twice very closely

**Review by beccabunny — APPROVED** [review]
It is worth noting that this person also uses 4 new outfits, which are also necessary given that the game weapons have hardpoints which wouldn't work here. It looks good to me, so I'm approving

File: data/remnant/remnant persons.txt [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+		forbearing mining surveillance unconstrained
+	phrase
+		phrase
+			"friendly author"
+	system
+		attributes "ember waste"
+	phrase
+		word
+			"I may be just a piece of rock, but"
+			"Even for a floating gathering of debris"
```
Comment (Amazinite) [inline data/remnant/remnant persons.txt:24]: This line doesn't really make sense to me with the potential follow-ups.
"Even for a floating gathering of debris you shouldn't buy anything the Syndicate sells."
Both these first two lines could use something a little extra to make them work out, really.
```suggestion
			"I may be just a piece of rock, but I know that"
			"Even for a floating gathering of debris I know that"
```

**Review by Amazinite — CHANGES_REQUESTED** [review]
Why a separate file? This should just be in the existing person ships file.

File: data/remnant/remnant persons.txt [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+			"transmission burst"
+		word
+			">"
+	ship "Shooting Star" "Prototype B3-CC4"
+		"never disabled"
+
+
+
+ship "Shooting Star"
+	sprite "ship/shooting star/shootingstar"
```
[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant persons.txt:110]: OK. Will do when I'm on desktop again.

[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant persons.txt:110]: Changed the sprite names.

File: data/remnant/remnant persons.txt [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+# Outfits
+
+outfit "Shooting Star Flare"
+	category "Guns"
+	thumbnail "outfit/unknown"
+	"gun ports" -1
+	weapon
+		"hardpoint sprite" "hardpoint/shooting star flare/ss-rays"
+			"frame time" 3
+	description "The beam projector of the Shooting Star, without it the flares would appear in the ship outline."
```
[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant persons.txt:169]: Descriptions removed.

**Comment by Zitchas** [comment]
Have B3-CC4 by themselves here doesn't make a ton of sense, admittedly. Next content (as in non-fix) PR is hopefully going to be my person, which, given previously discussed plans, makes more sense to have separate since it'll be a bit more variable. So I thought if I'm already considering having it separate for ease of dealing with it, it kind of makes sense to keep B3-CC4 in the same place, since B3-CC4 is also restricted to the Ember Waste.

The restriction is due to the fact that I've actually got some story planned with B3-CC4, which BeccaBunny liked the sounds of. 

The very bare-bones version is:
 - Remove the person ship from active use.
- Have the player find a disabled human ship in the Ember Waste. 
- Board it, and find a delusional Italian with a funny conversation. (Becca and I have worked out one or two elements of this)
- Bring the delusional Italian back to the Remnant, they take care of said person.
- Later find out that the person has recovered, and would much prefer to stay with the Remnant.
- Becomes an apprentice designer with Taely and use an event to add this person ship into active appearance.

Until the missions are actually added, though (probably a 0.9.15 thing), just having the person exist like this, which is what the end-state would be after the missions are done.

**Comment by Amazinite** [comment]
> Until the missions are actually added, though (probably a 0.9.15 thing), just having the person exist like this, which is what the end-state would be after the missions are done.

Problem with that is that I'm pretty sure you can't generate person ships through events. And as far as your person ship goes, I'd still want the person definition to be in the person file (regardless if where person ships might fit into any story anywhere), and you can put the ship definitions in the Remnant ships file since that's where they'd ultimately end up otherwise.

**Comment by Zitchas** [comment]
Re: Person in events: OK, well, maybe won't be able to do that part, then. Still planning/hoping to do the missions, though. Can events change the frequency or anything else to do with persons? Just changing the frequency from some massive number down to something reasonable could work too. Although that might influence the probabilities for everyone else, so might not do anything at all.

And the content has been moved into the persons.txt file.

**Review by Amazinite — APPROVED** [review]
The 2x PR needs updated.

**Comment by Zitchas** [comment]
It was just the name change from shootingstar to shooting star (and associated copyright file update), right? If so, that's done now.

**Comment by Zitchas** [comment]
Fixed the 2x ship and flare images, and uploaded a new (_v2) zip file on the google drive, so there's a new file link in the first post.

**Pull request merged**